packages/core-components/src/lib/unsorted/viz/core/_Sparkline.svelte

```diff
@@ -0,0 +1,391 @@
+					z: 2
+				}
+			],
+			series: [
+				{
+					type: seriesType,
+					triggerLineEvent: true,
+					label: {
+						show: false,
+						position: 'top',
+						padding: 0,
+						fontSize: 9
+					},
+					labelLayout: {
+						hideOverlap: true
+					},
+					connectNulls: false,
+					emphasis: {
+						disabled: true
+					},
+					lineStyle: {
+						width: 1,
+						type: 'solid',
+						color: color ?? uiColours.grey400
+					},
+					areaStyle: {
+						color:
+							type === 'area'
+								? color
+									? chroma(color).brighten(1.5).hex()
+									: uiColours.grey300
+								: 'transparent'
+					},
+					itemStyle: {
+						color: color ?? uiColours.grey400
+					},
+					showSymbol: false,
+					symbol: 'circle',
+					symbolSize: 0,
+					step: false,
+					name: 'sparkline',
+					data: sparklineData,
+					yAxisIndex: 0
+				}
+			],
+			animation: false,
+			graphic: {
+				id: 'horiz-axis-title',
+				type: 'text',
+				style: {
+					text: '',
+					textAlign: 'right',
+					fill: 'hsla(212, 10%, 53%, 1)'
+				},
+				cursor: 'auto',
+				right: '3%',
+				top: null,
+				bottom: '2%'
+			}
+		};
+
+		if (!browser) {
+			// SSR-specific initialization
+			const tempChart = init(null, 'evidence-light', {
+				ssr: true,
+				renderer: 'svg',
+				height: height,
+				width: width
+			});
+			tempChart.setOption(config);
+			staticSVGSSR = tempChart.renderToSVGString();
+			tempChart.dispose(); // Dispose instance after generating SVG
+		}
+
+		// Initialize the static SVG
+		onMount(() => {
+			if (!interactive) {
+				// Generate static SVG for non-interactive mode
+				const offscreenContainer = document.createElement('div');
+				offscreenContainer.style.width = width + 'px';
+				offscreenContainer.style.height = height + 'px';
+				const tempChart = init(offscreenContainer, null, { renderer: 'svg', height, width });
+				tempChart.setOption(config);
+				staticSVG = tempChart.renderToSVGString();
+				tempChart.dispose();
+			} else {
+				initializeChart();
+			}
+		});
+
+		if (chartContainer && interactive && !chartInstance) {
+			initializeChart();
+		}
+
+		// Cleanup
+		onDestroy(() => {
+			if (chartInstance) {
+				chartInstance.dispose();
+			}
+		});
+	} catch (e) {
+		error = e;
+		const setTextRed = '\x1b[31m%s\x1b[0m';
+		console.error(setTextRed, `Error in Sparkline: ${error.message}`);
+		if (strictBuild) {
+			throw error;
+		}
+	}
+
+	$: data, config;
+</script>
+
+{#if error}
+	<ValueError {error} />
+{:else if !browser}
+	<div
+		class="inline-block align-baseline border-blue-600 border-0"
+		style="width: {width}px; height: {height}px;"
+	>
+		{@html staticSVGSSR}
+	</div>
+{:else if !interactive}
+	<div
+		class="inline-block align-baseline border-blue-600 border-0"
+		style="width: {width}px; height: {height}px;"
+	>
+		{@html staticSVG}
+	</div>
+{:else}
+	<div
+		bind:this={chartContainer}
+		class="z-50 inline-block align-baseline overflow-visible border-blue-400 border-0"
```

Comment:
The sparklines are visible through the mobile sidebar, suspect the z-50 is to blame

Reply:
Nice, good catch